doc/rst/quick.rst

```diff
@@ -21,7 +21,7 @@ Quick Start
 
    #. Download the **XMOS XTC Tools** from: http://www.xmos.com/software-tools and install.
 
-      The firmware should be compiled using a specific version of the tools. Make sure that you download the correct version of the tools.
+      The minimum required XTC Tools version for compiling USB Audio applications is 15.3.0. Make sure that you download the correct version of the tools.
```

Comment:
Tell user to check readme for tools version number as we'll keep that up to date (soon to be automated)